composer.json

```diff
@@ -43,7 +43,7 @@
 	},
 	"require-dev": {
 		"nextcloud/ocp": "dev-stable26",
-		"psalm/phar": "^5.25.0",
+		"psalm/phar": "^5.9.0",
```

Comment:
This is wrong, you already have good version of psalm. No need to downgrade it, anything >=5.9.0 is fine so 5.25.0 is fine too.

Reply:
ugh I'm so confused, of course 5.9 is lower than 5.25 :sweat: